aidbox_python_sdk/sdk.py

```diff
@@ -151,6 +152,7 @@ def operation(  # noqa: PLR0913
         access_policy=None,
         request_schema=None,
         timeout=None,
+        compliance_params: Optional[Dict[str, Any]] = None,
```

Comment:
Let's name it `compliance`.

Use https://peps.python.org/pep-0589/ to define required keys in the dict.